exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestAnalyze.java

```diff
@@ -480,6 +490,24 @@ public void testHistogramWithColumnsWithAllNulls() throws Exception {
     }
   }
 
+
+  @Test
+  public void testHistogramWithBetweenPredicate() throws Exception {
+    try {
+      test("ALTER SESSION SET `planner.slice_target` = 1");
+      test("ALTER SESSION SET `store.format` = 'parquet'");
+      test("create table dfs.tmp.orders2 as select * from cp.`tpch/orders.parquet`");
+      test("analyze table dfs.tmp.orders2 compute statistics");
+      test("alter session set `planner.statistics.use` = true");
+
+      String query = "select 1 from dfs.tmp.orders2 o where o.o_orderdate >= date '1996-10-01' and o.o_orderdate < date '1996-10-01' + interval '3' month";
```

Comment:
I do not see the `BETWEEN` clause in this query.

Reply:
I changed the test name to use 'Interval' instead of 'Between' since that's the main purpose of this test.  There are 2 other tests I added which do use `BETWEEN` clause in `testHistogramWithDataTypes1`.